src/classes/ElevateCaptureGroupService.cls

```diff
@@ -48,6 +48,37 @@ public virtual with sharing class ElevateCaptureGroupService {
         return new ElevateAuthorizedGift(tokenizedGift, authorizedGiftResponse);
     }
 
+    public ElevateCaptureGroup getCaptureGroup() {
+        UTIL_Http.Response response = new UTIL_Http.RequestService().sendRequest(
+                createRequest()
+        );
+        ElevateCaptureGroupResponse captureGroupResponse = (ElevateCaptureGroupResponse)JSON.deserialize(
+                response.body, ElevateCaptureGroupResponse.class);
+        return new ElevateCaptureGroup(captureGroupResponse);
+    }
+
+    private HttpRequest addRequest(ElevateTokenizedGift tokenizedGift, String groupId) {
+        return new PS_Request.Builder()
+                .withMethod(UTIL_Http.Method.POST)
+                .withCaptureGroupId(groupId)
+                .withEndpoint(PS_Request.ElevateEndpoint.ADD_TO_CAPTURE_GROUP)
+                .withBody(JSON.serialize(tokenizedGift))
+                .build();
+    }
+
+    public List<ElevateCaptureGroupResponse> capture(Set<String> captureGroupIds) {
+        List<ElevateCaptureGroupResponse> elevateCaptureGroupResponses = new List<ElevateCaptureGroupResponse>();
+        for (String elevateBatchId : captureGroupIds) {
+            UTIL_Http.Response response = new UTIL_Http.RequestService().sendRequest(
```

Comment:
This is a probably a veryyyyy minor edge case but I figured I'd raise it just in case. Given that we could be making multiple callouts to the elevate "capture" endpoint when a BGE batch contains multiple capture groups, is there any concern that we might hit governor limits?